Specs for contact (#257)
PrestaShop/prestashop-specs: content/1.7/back-office/shop-parameters/contact/add-edit-contacts.md

---
title: Add / edit contact
weight: 1
---
# Add / edit contact

> ![Contact](/img/contacts.png)

Reviewer: This image doesn't exist

> Click the "Add new contact" button or edit an existingg contact to access the contact creation form.

Reviewer: ```suggestion
```

While adding or editing a contact, the following fields / toggles / parameters are displayed:

 1) Title

> This field is mandatory and can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.

Reviewer: Please specify what happens if the form is saved and if this field isn't filled

> Title is the name of the contact: either a person's name or the name of the service itself. Has an explanation below the field written: "Contact name (e.g. Customer Support)."

Reviewer: ```suggestion
The title is the name of the contact. For example, it can be either a person's name or the name of the service itself. There is a help-text under this field: "Contact name (e.g. Customer Support)."
```

 
 2) Email address

> Email address input, has an explanation below it: "Emails will be sent to this address."."

Reviewer: ```suggestion
There is a help-text under this field: "Emails will be sent to this address."."
```

>  3) Save messages?

Reviewer: ```suggestion
 
 3) Save messages?
```

This is a toggle switch, can be choosed as "yes" or "no". It is a mandatory toggle. Has an explanation below it: If enabled, all messages will be saved in the "Customer Service" page under the "Customer" menu."

 4) Description

This field is mandatory and can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.

 5) Shop association

> The shop association block is displayed when the multistore is enabled and if several shops are configured. This field is mandatory.

Reviewer: ```suggestion
The shop association block is mandatory and is displayed when the multistore is enabled and if several shops are configured.
```

Reviewer: Please specify what happens if the form is saved and if this field isn't filled

---
title: Add / edit contact
weight: 1
---
# Add / edit contact

> ![Contact](/img/contacts.png)

Reviewer: ```suggestion
![Contact](static/img/contacts.png)
```

> ![Contact](/img/contacts.png)

Reviewer: ```suggestion
![Contact](/static/img/contacts.png)
```

While adding or editing a contact, the following fields / toggles / parameters are displayed:

 1) Title

This field is mandatory and can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
The title is the name of the contact. For example, it can be either a person's name or the name of the service itself. There is a help-text under this field: "Contact name (e.g. Customer Support)."

> If the field is left empty, and when trying to save, shows the message below the field: "The field title is required at least in your default language."

Reviewer: ```suggestion
If the save button is pressed and if this field is empty for the default language, the following error message is displayed: "The field title is required at least in your default language."
```

 
 2) Email address

There is a help-text under this field: "Emails will be sent to this address.".
 
 3) Save messages?

This toggle is mandatory and allows to choose if the messages are saved or not in the Customer Service page. By default, this toggle is enabled. There is a help-text under this toggle: "If enabled, all messages will be saved in the "Customer Service" page under the "Customer" menu."

 4) Description

This field can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.

 5) Shop association

The shop association block is mandatory and is displayed when the multistore is enabled and if several shops are configured.

> If none of the shops are selected, does not save the the contact, shows the message: "The "Shop association" field is required."

Reviewer: ```
If the save button is pressed and if no shop is selected, the following error message is displayed: "The "Shop association" field is required." and the contact is not saved.
```

> If none of the shops are selected, does not save the the contact, shows the message: "The "Shop association" field is required."

Reviewer: ```suggestion
If the save button is pressed and if no shop is selected, the following error message is displayed: "The "Shop association" field is required." and the contact is not saved.
```
